### ISSUE TITLE
Documentation errors in 'The Java 8 / Jetty 9.3 Runtime'

### ISSUE THREAD

**Issue by gregw** [GoogleCloudPlatform/appengine-java-vm-runtime#188]
Not sure if this is the right place to raise an issue on this [docomentation](https://cloud.google.com/appengine/docs/managed-vms/java/dev-jetty9#customize), but I will do so to at least capture to issues.

To use a custom `Dockerfile`, the runtime in the `app.yaml` file must be changed from `java` to `custom`

The documentation says that if you wish to use a custom Dockerfile, you must build the war and put it in the source directory next to app.yml.    This is not the case, rather the war is already build for you and is placed in target/appengine-staging.   A dockerfile as follows may be used:
````
FROM gcr.io/google_appengine/jetty9
ADD projectname-version.war $JETTY_BASE/webapps/root.war
````
where projectname and version are from maven (it would be really good if maven property expansion could be done on the Dockerfile when copied to target!!!)

**Comment by lesv** [comment]
Not sure where the war thing came in. --- It wasn't what I reviewed.

I'll look into this.

**Comment by lesv** [comment]
You can have Maven Property expansion in a Dockerfile, it should do it during resource copy phase.

I've reported this internally.

**Comment by gregw** [comment]
Uffa!   I've finally got an example running at it was a lot harder that I had thought.... partly due to my own stupidity, in part bad jetty doco, but also a miss-match with the jetty module system and clustered session management, as well as the google doc that needs to be improved.

Anyway, I have an example webapp, which I propose to create a PR against  java-docs-samples/managed_vms/jetty9-cloud-session... standby

I also raised: https://github.com/eclipse/jetty.project/issues/378 and https://github.com/eclipse/jetty.project/issues/380 that will make things simpler in future.

**Comment by gregw** [comment]
@aozarov This is the issue under which I prepared an example of a non-compat shared session manager, but that really was for  #183.   I had prepared PR https://github.com/GoogleCloudPlatform/java-docs-samples/pull/123, but it was not polished enough to be merged, so I closed it.

I think this issue is too general and should be closed, with more specific issues created for any documentation required.       Non compat session clustering work should be done under #183
